### ISSUE TITLE
DiceLoss

### ISSUE THREAD

**Issue by jdavidd** [wolny/pytorch-3dunet#25]
Hello.
I want to use GeneralisedDIceLoss, class.
My network output  a tensor of [30, 2, 128, 128] - 2 channels map, and target is [30, 1, 128, 128] 1 channel, which works with CrossEntropy, but how can I use you DiceLoss? It's ok to make a max on output and have output [30, 1, 128, 128]?
I have seen this issue, already but you said you solved the whole arhitecture, I only need the loss function. Please help.

**Comment by wolny** [comment]
Hi @jdavidd.
All loss functions are implemented in the following file: https://github.com/wolny/pytorch-3dunet/blob/master/unet3d/losses.py#L75, so feel free to use the code from there and adapt it to your needs

**Comment by jdavidd** [comment]
I saw that the code is there =)). I have asked some help how can i modify your function.... Can you help me?
